modules/AWS/rds/locals.tf

```diff
@@ -1,5 +1,7 @@
 locals {
-  product_db_name    = var.product_db_name == null ? var.product : var.product_db_name
+  product_db_name   = var.product
```

Comment:
nit: is it necessary to create a new variable for it? but I guess it clearly states that it's the db name.